tiledb/sm/query/writer.cc

```diff
@@ -2395,10 +2407,11 @@ Status Writer::prepare_full_tiles_fixed(
   auto it = buffers_.find(name);
   auto buffer = (unsigned char*)it->second.buffer_;
   auto buffer_validity = (unsigned char*)it->second.validity_vector_.buffer();
-  auto buffer_size = it->second.buffer_size_;
+  auto buffer_size =
```

Comment:
In the context of fixed attributes, the first buffer contains values. This path is for fixed attributes, but is now treated as if it contains offsets. I think this line should be reverted.